nasl/nasl.c

```diff
@@ -289,8 +283,6 @@ main (int argc, char **argv)
       fprintf (stderr, "** WARNING : packet forgery will not work\n");
       fprintf (stderr, "** as NASL is not running as root\n");
     }
-  signal (SIGINT, sighandler);
-  signal (SIGTERM, sighandler);
```

Comment:
I think that the intention here was to finish the program in a normal way (with exit(0)) even when the user sends a kill or interrupts with `ctrl + c`. 
openvas-nasl sometimes forks. I an not sure if it would be the same for child processes to kill or interrupt the parent process than an exit(). For sure we are changing the exit code from 0 to the respectively 2 and 15.
Do we want to change this behavior? TBH I don't know what to do here . Feel free to change it if you want.